src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -249,22 +249,6 @@ pub enum ParseResult<T> {
 pub type NamedParseResult = ParseResult<HashMap<Ident, Rc<NamedMatch>>>;
 pub type PositionalParseResult = ParseResult<Vec<Rc<NamedMatch>>>;
 
-pub fn parse_or_else(sess: &ParseSess,
```

Comment:
Maybe this function could be changed to take in a `substitute_span: Span` and have the `DUMMY_SP` comparison internally?